src/Content/Item.php

```diff
@@ -432,6 +432,8 @@ public function photoMenu(array $item, string $formSecurityToken): string
 				$this->l10n->t('Network Posts') => $posts_link,
 				$this->l10n->t('View Contact') => $contact_url,
 				$this->l10n->t('Send PM') => $pm_url,
+				$this->l10n->t('Mention') => $mention_url,
+				$this->l10n->t('Post to group') => $mention_url,
```

Comment:
Why is it the same URL in both cases?

Reply:
I must confess, i don't know, what happens here... 
Have orientated on the code... but now i removed this lines completely, and it works also.

Can you please explain, what this function does?